clueless/models.py

```diff
@@ -28,8 +42,9 @@ class Player(models.Model):
     Represents a player in the context of a clueless game.  Ties back to Django user
     """
     user = models.ForeignKey(User)
-    currentSpace = models.ForeignKey(Space)
-    currentGame = models.ForeignKey('Game') # game not defined yet, using string as lazy lookup
+    currentSpace = models.ForeignKey(Space, blank=True)
```

Comment:
There will be a period of time between when a game is started, and everyone else joins the game.  At this point it's not even guaranteed that the user has chosen their character yet.  Given that, I'd rather leave it blank initially and then fill it in when the game really get's started (which maybe we need a different method for).